### PR TITLE
TypeScript Typings

### PR DESCRIPTION
**Pull request by igorkamyshev**
Hello! Great lib. I tried to find typings for it, but couldn't, then — I just wrote it. Please, review my PR and give me your feedback.

### CODE REVIEW

**Comment by kenmaca** [comment]
Hi Igor! Thanks for the contribution!

The only feedback I'd have for you is `Props` should be named `IProps` instead. `tslint` will probably complain for most people.

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+interface Props {
```
Comment (kenmaca) [inline index.d.ts]: Should probably be named `IProps` instead to avoid `tslint` from complaining.

**Review by kenmaca — APPROVED** [review]

**Comment by igorkamyshev** [comment]
Done

**Review by kenmaca — APPROVED** [review]
🎉👏

**Comment by igorkamyshev** [comment]
Can you merge it?

**Pull request merged**